supertokens_python/recipe/emailverification/utils.py

```diff
@@ -57,26 +44,26 @@ def __init__(
         self.apis = apis
 
 
+MODE_TYPE = Literal["REQUIRED", "OPTIONAL"]
```

Comment:
does this do a compile error if the user doesn't give one of these two (even if they do not use a linter). Cause if not, we should have input validation to make sure that the value is indeed one of these and if not, throw an appropriate error.

Reply:
It doesn't give compile error. 

Added validation in `validate_and_normalise_user_input` of the ev recipe. Error message would be `Email Verification recipe mode must be one of 'REQUIRED' or 'OPTIONAL'`.